templates/minimal/README.md

```diff
@@ -37,11 +37,25 @@
 - 🤏 This template is a minimal (in terms of complexity and the number of components)
 template for building a blockchain node.
 
+
+- 🔧 To simplify the process of using Zombienet with Omni Node, we've added a pre-configured development
+chain spec (dev_chain_spec.json) to the minimal template. Additionally, the zombienet-omni-node.toml file
+has been updated to include a valid chain spec path pointing to dev_chain_spec.json
+
+
 - 🔧 Its runtime is configured with a single custom pallet as a starting point, and a handful of ready-made pallets
 such as a [Balances pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_balances/index.html).
 
+
 - 👤 The template has no consensus configured - it is best for experimenting with a single node network.
 
+For the most accurate chain spec generation,  `staging-chain-spec-builder` or other mentioned binaries and runtimes,
+please try to build them based on the Polkadot-sdk most updated master branch an example:
+  ```bash
+  cargo install --path substrate/bin/utils/chain-spec-builder
+  ```
```

Comment:
This is a good note, but I wouldn't add it anymore. Latest `staging-chain-spec-builder` is good enough for what this readme needs: https://crates.io/crates/staging-chain-spec-builder

Reply:
We also need `--locked` here. Please doublecheck all the install commands. `cargo install` by default does not use the lockfile.